common/blast-query-parser/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+plugins {
+  kotlin("jvm") version "1.7.20"
+  id("org.jetbrains.dokka") version "1.7.10"
+  `maven-publish`
+}
+
+group = "org.veupathdb.lib"
+version = "1.0-SNAPSHOT"
+
+dependencies {
+  implementation("org.veupathdb.lib:mblast-utils:1.0-SNAPSHOT") { isChanging = true }
```

Comment:
Just checking, is this purposely SNAPSHOT for now while in the midst of the refactor/re-write?

Reply:
Yep, it will remain snapshot until I am sure there are no major changes needed to it.  It should get set to v1.0.0 when the project is closer to completion.